### DISCUSSION TITLE
getting error while running open5gs docker

### DISCUSSION THREAD

**Discussion by rajugbv** [srsran/srsRAN_Project#420] [General Help]
Hi everyone,

I am encountering an error while running the Docker command {docker-compose up --build 5gc}. Please let me know if I need to make any configurations
aimslab@aimslab-ThinkPad-P16s-Gen-1:~/new/srsRAN_Project/docker$ sudo docker-compose up 
[+] Running 5/0
 ✔ Container metrics_server  Running                                                                                         0.0s 
 ✔ Container influxdb        Running                                                                                         0.0s 
 ✔ Container open5gs_5gc     Running                                                                                         0.0s 
 ✔ Container grafana         Running                                                                                         0.0s 
 ✔ Container srsran_gnb      Created                                                                                         0.0s 
Attaching to grafana, influxdb, metrics_server, open5gs_5gc, srsran_gnb
open5gs_5gc     | 01/22 18:13:30.315: [amf] INFO: gNB-N2 accepted[10.53.1.3]:58428 in ng-path module (../src/amf/ngap-sctp.c:113)
open5gs_5gc     | 01/22 18:13:30.315: [amf] INFO: gNB-N2 accepted[10.53.1.3] in master_sm module (../src/amf/amf-sm.c:706)
open5gs_5gc     | 01/22 18:13:30.315: [amf] INFO: [Added] Number of gNBs is now 1 (../src/amf/context.c:1034)
open5gs_5gc     | 01/22 18:13:30.315: [amf] INFO: gNB-N2[10.53.1.3] max_num_of_ostreams : 30 (../src/amf/amf-sm.c:745)
srsran_gnb      | [INFO] [UHD] linux; GNU C++ version 11.2.0; Boost_107400; UHD_4.1.0.5-3
srsran_gnb      | [INFO] [LOGGING] Fastpath logging disabled at runtime.
srsran_gnb      | [INFO] [B200] Detected Device: B210
srsran_gnb      | [INFO] [B200] Operating over USB 3.
srsran_gnb      | [INFO] [B200] Initialize CODEC control...
srsran_gnb      | [INFO] [B200] Initialize Radio control...
srsran_gnb      | [INFO] [B200] Performing register loopback test... 
srsran_gnb      | [INFO] [B200] Register loopback test passed
srsran_gnb      | [INFO] [B200] Performing register loopback test... 
srsran_gnb      | [INFO] [B200] Register loopback test passed
srsran_gnb      | [INFO] [B200] Setting master clock rate selection to 'automatic'.
srsran_gnb      | [INFO] [B200] Asking for clock rate 16.000000 MHz... 
srsran_gnb      | [INFO] [B200] Actually got clock rate 16.000000 MHz.
srsran_gnb      | [INFO] [MULTI_USRP] Setting master clock rate selection to 'manual'.
srsran_gnb      | [INFO] [B200] Asking for clock rate 23.040000 MHz... 
srsran_gnb      | [INFO] [B200] Actually got clock rate 23.040000 MHz.
srsran_gnb      | srsGNB ERROR: Unable to create radio session.
srsran_gnb      | 
srsran_gnb      | The PRACH detector will not meet the performance requirements with the configuration {Format 0, ZCZ 0, SCS 1.25kHz, Rx ports 1}.
srsran_gnb      | Lower PHY in quad executor mode.
srsran_gnb      | 
srsran_gnb      | --== srsRAN gNB (commit ) ==--
srsran_gnb      | 
srsran_gnb      | Connecting to AMF on 10.53.1.2:38412
srsran_gnb      | Available radio types: uhd.
srsran_gnb      | Making USRP object with args 'type=b200'
srsran_gnb      | Could not lock reference clock source. Sensor: ref_locked=false.
open5gs_5gc     | 01/22 18:13:32.986: [amf] INFO: gNB-N2[10.53.1.3] connection refused!!! (../src/amf/amf-sm.c:758)
open5gs_5gc     | 01/22 18:13:32.986: [amf] INFO: [Removed] Number of gNBs is now 0 (../src/amf/context.c:1061)

**Comment by pgawlowicz** [comment]
You said there is an error when running `docker-compose up --build 5gc`, but then you show that you run `sudo docker-compose up`. Is it correct?

**Reply by rajugbv** [reply]
@pgawlowicz Sorry for that. I think I have posted the wrong log. Let me post the correct log. While running the command 'docker-compose up --build 5gc', it reaches the following warning messages:
### open5gs_5gc | 01/24 21:08:49.618: [smf] WARNING: PFCP[REQ] has already been associated [127.0.0.7]:8805 (../src/smf/pfcp-sm.c:217)
### open5gs_5gc | 01/24 21:08:49.618: [upf] WARNING: PFCP[RSP] has already been associated [127.0.0.4]:8805 (../src/upf/pfcp-sm.c:213)

After these warnings, it gets stuck for a while, and eventually, I am getting a segmentation error.


aimslab@aimslab-ThinkPad-P16s-Gen-1:~/srsRAN_Project/docker$ sudo docker-compose up --build 5gc
[sudo] password for aimslab: 
[+] Building 0.1s (16/16) FINISHED                               docker:default
 => [5gc internal] load build definition from Dockerfile                   0.0s
 => => transferring dockerfile: 3.11kB                                     0.0s
 => [5gc internal] load .dockerignore                                      0.0s
 => => transferring context: 2B                                            0.0s
 => [5gc internal] load metadata for docker.io/library/ubuntu:22.04        0.0s
 => [5gc base 1/8] FROM docker.io/library/ubuntu:22.04                     0.0s
 => [5gc internal] load build context                                      0.0s
 => => transferring context: 14.70kB                                       0.0s
 => CACHED [5gc base 2/8] RUN DEBIAN_FRONTEND=noninteractive apt-get upda  0.0s
 => CACHED [5gc base 3/8] RUN DEBIAN_FRONTEND=noninteractive apt-get upda  0.0s
 => CACHED [5gc base 4/8] RUN DEBIAN_FRONTEND=noninteractive apt-get upda  0.0s
 => CACHED [5gc base 5/8] RUN echo v2.6.1 > ./open5gsversion               0.0s
 => CACHED [5gc base 6/8] RUN git clone --depth 1 --branch $(cat ./open5g  0.0s
 => CACHED [5gc base 7/8] RUN curl -fsSL https://deb.nodesource.com/setup  0.0s
 => CACHED [5gc base 8/8] RUN python3 -m pip install pymongo click pyrout  0.0s
 => CACHED [5gc open5gs 1/3] WORKDIR /open5gs                              0.0s
 => CACHED [5gc open5gs 2/3] COPY open5gs-5gc.yml open5gs-5gc.yml.in       0.0s
 => CACHED [5gc open5gs 3/3] COPY open5gs_entrypoint.sh add_users.py setu  0.0s
 => [5gc] exporting to image                                               0.0s
 => => exporting layers                                                    0.0s
 => => writing image sha256:bcf1f59e3131e9a12bc8e9f9c7d681b994bbfba7ee882  0.0s
 => => naming to docker.io/library/docker-5gc                              0.0s
[+] Running 1/0
 ✔ Container open5gs_5gc  Created                                          0.0s 
Attaching to open5gs_5gc
open5gs_5gc  | nc: connect to 127.0.0.1 port 27017 (tcp) failed: Connection refused
open5gs_5gc  | waiting for mongodb
open5gs_5gc  | {"t":{"$date":"2024-01-24T21:08:44.403Z"},"s":"I",  "c":"CONTROL",  "id":20697,   "ctx":"-","msg":"Renamed existing log file","attr":{"oldLogPath":"/tmp/mongodb.log","newLogPath":"/tmp/mongodb.log.2024-01-24T21-08-44"}}
open5gs_5gc  | 
open5gs_5gc  | > open5gs@2.6.1 dev
open5gs_5gc  | > node server/index.js
open5gs_5gc  | 
open5gs_5gc  | > Using external babel configuration
open5gs_5gc  | > Location: "/open5gs/webui/.babelrc"
open5gs_5gc  | Connection to 127.0.0.1 27017 port [tcp/*] succeeded!
open5gs_5gc  |  DONE  Compiled successfully in 593ms9:08:45 PM
open5gs_5gc  | 
open5gs_5gc  | Mongoose: accounts.count({}, {})
open5gs_5gc  | > Ready on http://16b1fd582042:3000
open5gs_5gc  | Mongoose: subscribers.createIndex({ imsi: 1 }, { unique: true, background: true })
open5gs_5gc  | Mongoose: accounts.createIndex({ username: 1 }, { unique: true, background: true })
open5gs_5gc  | SUBSCRIBER_DB=001010123456780,00112233445566778899aabbccddeeff,opc,63bfa50ee6523365ff14c1f45f88737d,8000,9,10.45.1.2
open5gs_5gc  | Reading subscriber data from cmd.
open5gs_5gc  | UE (IMSI=001010123456780) already exists, updating it.
open5gs_5gc  | Attempting to update IMSI 001010123456780
open5gs_5gc  | UpdateResult({'n': 1, 'nModified': 0, 'ok': 1.0, 'updatedExisting': True}, acknowledged=True)
open5gs_5gc  | True
open5gs_5gc  | Open5GS daemon v2.6.1
open5gs_5gc  | 
open5gs_5gc  | 01/24 21:08:46.995: [app] INFO: Configuration: 'open5gs-5gc.yml' (../lib/app/ogs-init.c:126)
open5gs_5gc  | Open5GS daemon v2.6.1
open5gs_5gc  | 
open5gs_5gc  | 01/24 21:08:47.004: [app] INFO: Configuration: 'open5gs-5gc.yml' (../lib/app/ogs-init.c:126)
open5gs_5gc  | 01/24 21:08:47.006: [sbi] INFO: nghttp2_server() [http://127.0.0.10]:7777 (../lib/sbi/nghttp2-server.c:395)
open5gs_5gc  | 01/24 21:08:47.006: [app] INFO: NRF initialize...done (../src/nrf/app.c:31)
open5gs_5gc  | Open5GS daemon v2.6.1
open5gs_5gc  | 
open5gs_5gc  | 01/24 21:08:47.060: [app] INFO: Configuration: 'open5gs-5gc.yml' (../lib/app/ogs-init.c:126)
open5gs_5gc  | 01/24 21:08:47.063: [sbi] INFO: nghttp2_server() [http://127.0.1.10]:7777 (../lib/sbi/nghttp2-server.c:395)
open5gs_5gc  | 01/24 21:08:47.063: [app] INFO: SCP initialize...done (../src/scp/app.c:31)
open5gs_5gc  | 01/24 21:08:47.064: [nrf] INFO: [c390a4fa-bafc-41ee-915a-39662b03c7f6] NF registered [Heartbeat:10s] (../src/nrf/nf-sm.c:190)
open5gs_5gc  | 01/24 21:08:47.065: [sbi] INFO: [c390a4fa-bafc-41ee-915a-39662b03c7f6] NF registered [Heartbeat:10s] (../lib/sbi/nf-sm.c:214)
open5gs_5gc  | Open5GS daemon v2.6.1
open5gs_5gc  | 
open5gs_5gc  | 01/24 21:08:47.110: [app] INFO: Configuration: 'open5gs-5gc.yml' (../lib/app/ogs-init.c:126)
open5gs_5gc  | 01/24 21:08:47.114: [pfcp] INFO: pfcp_server() [127.0.0.7]:8805 (../lib/pfcp/path.c:30)
open5gs_5gc  | 01/24 21:08:47.114: [pfcp] INFO: ogs_pfcp_connect() [127.0.0.4]:8805 (../lib/pfcp/path.c:61)
open5gs_5gc  | 01/24 21:08:47.115: [gtp] INFO: gtp_server() [10.53.1.2]:2152 (../lib/gtp/path.c:30)
open5gs_5gc  | 01/24 21:08:47.115: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
open5gs_5gc  | Open5GS daemon v2.6.1
open5gs_5gc  | 
open5gs_5gc  | 01/24 21:08:47.166: [app] INFO: Configuration: 'open5gs-5gc.yml' (../lib/app/ogs-init.c:126)
open5gs_5gc  | 01/24 21:08:47.200: [gtp] INFO: gtp_server() [127.0.0.4]:2123 (../lib/gtp/path.c:30)
open5gs_5gc  | 01/24 21:08:47.200: [gtp] INFO: gtp_server() [127.0.0.4]:2152 (../lib/gtp/path.c:30)
open5gs_5gc  | 01/24 21:08:47.200: [pfcp] INFO: pfcp_server() [127.0.0.4]:8805 (../lib/pfcp/path.c:30)
open5gs_5gc  | 01/24 21:08:47.200: [pfcp] INFO: ogs_pfcp_connect() [127.0.0.7]:8805 (../lib/pfcp/path.c:61)
open5gs_5gc  | 01/24 21:08:47.200: [sbi] INFO: NF Service [nsmf-pdusession] (../lib/sbi/context.c:1436)
open5gs_5gc  | 01/24 21:08:47.201: [upf] INFO: PFCP associated [127.0.0.4]:8805 (../src/upf/pfcp-sm.c:173)
open5gs_5gc  | 01/24 21:08:47.201: [sbi] INFO: nghttp2_server() [http://127.0.0.4]:7777 (../lib/sbi/nghttp2-server.c:395)
open5gs_5gc  | 01/24 21:08:47.201: [app] INFO: SMF initialize...done (../src/smf/app.c:31)
open5gs_5gc  | 01/24 21:08:47.201: [smf] INFO: PFCP associated [127.0.0.7]:8805 (../src/smf/pfcp-sm.c:174)
open5gs_5gc  | 01/24 21:08:47.202: [nrf] INFO: [c3a12eb0-bafc-41ee-9d94-bb7137604f52] NF registered [Heartbeat:10s] (../src/nrf/nf-sm.c:190)
open5gs_5gc  | 01/24 21:08:47.202: [sbi] INFO: [c3a12eb0-bafc-41ee-9d94-bb7137604f52] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc  | 01/24 21:08:47.202: [sbi] INFO: [c3a12eb0-bafc-41ee-9d94-bb7137604f52] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc  | 01/24 21:08:47.202: [sbi] INFO: [c3a12eb0-bafc-41ee-9d94-bb7137604f52] NF registered [Heartbeat:10s] (../lib/sbi/nf-sm.c:214)
open5gs_5gc  | Open5GS daemon v2.6.1
open5gs_5gc  | 
open5gs_5gc  | 01/24 21:08:47.205: [app] INFO: Configuration: 'open5gs-5gc.yml' (../lib/app/ogs-init.c:126)
open5gs_5gc  | 01/24 21:08:47.209: [sbi] INFO: NF Service [namf-comm] (../lib/sbi/context.c:1436)
open5gs_5gc  | 01/24 21:08:47.210: [sbi] INFO: nghttp2_server() [http://127.0.0.5]:7777 (../lib/sbi/nghttp2-server.c:395)
open5gs_5gc  | 01/24 21:08:47.210: [amf] INFO: ngap_server() [10.53.1.2]:38412 (../src/amf/ngap-sctp.c:61)
open5gs_5gc  | 01/24 21:08:47.210: [sctp] INFO: AMF initialize...done (../src/amf/app.c:33)
open5gs_5gc  | 01/24 21:08:47.211: [nrf] INFO: [c3a6be52-bafc-41ee-b0f9-9116853993e6] NF registered [Heartbeat:10s] (../src/nrf/nf-sm.c:190)
open5gs_5gc  | 01/24 21:08:47.211: [sbi] INFO: [c3a6be52-bafc-41ee-b0f9-9116853993e6] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc  | 01/24 21:08:47.211: [sbi] INFO: [c3a6be52-bafc-41ee-b0f9-9116853993e6] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc  | 01/24 21:08:47.211: [sbi] INFO: [c3a6be52-bafc-41ee-b0f9-9116853993e6] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc  | 01/24 21:08:47.211: [sbi] INFO: [c3a6be52-bafc-41ee-b0f9-9116853993e6] NF registered [Heartbeat:10s] (../lib/sbi/nf-sm.c:214)
open5gs_5gc  | 01/24 21:08:47.211: [sbi] INFO: [c3a6be52-bafc-41ee-b0f9-9116853993e6] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc  | Open5GS daemon v2.6.1
open5gs_5gc  | 
open5gs_5gc  | 01/24 21:08:47.260: [app] INFO: Configuration: 'open5gs-5gc.yml' (../lib/app/ogs-init.c:126)
open5gs_5gc  | 01/24 21:08:47.262: [sbi] INFO: NF Service [nausf-auth] (../lib/sbi/context.c:1436)
open5gs_5gc  | 01/24 21:08:47.262: [sbi] INFO: nghttp2_server() [http://127.0.0.11]:7777 (../lib/sbi/nghttp2-server.c:395)
open5gs_5gc  | 01/24 21:08:47.262: [app] INFO: AUSF initialize...done (../src/ausf/app.c:31)
open5gs_5gc  | 01/24 21:08:47.263: [nrf] INFO: [c3af1502-bafc-41ee-865f-0ff49501ec81] NF registered [Heartbeat:10s] (../src/nrf/nf-sm.c:190)
open5gs_5gc  | 01/24 21:08:47.263: [sbi] INFO: [c3af1502-bafc-41ee-865f-0ff49501ec81] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc  | 01/24 21:08:47.263: [sbi] INFO: [c3af1502-bafc-41ee-865f-0ff49501ec81] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc  | 01/24 21:08:47.263: [sbi] INFO: [c3af1502-bafc-41ee-865f-0ff49501ec81] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc  | 01/24 21:08:47.263: [sbi] INFO: [c3af1502-bafc-41ee-865f-0ff49501ec81] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc  | 01/24 21:08:47.263: [sbi] INFO: [c3af1502-bafc-41ee-865f-0ff49501ec81] NF registered [Heartbeat:10s] (../lib/sbi/nf-sm.c:214)
open5gs_5gc  | Open5GS daemon v2.6.1
open5gs_5gc  | 
open5gs_5gc  | 01/24 21:08:47.310: [app] INFO: Configuration: 'open5gs-5gc.yml' (../lib/app/ogs-init.c:126)
open5gs_5gc  | 01/24 21:08:47.311: [sbi] INFO: NF Service [nudm-ueau] (../lib/sbi/context.c:1436)
open5gs_5gc  | 01/24 21:08:47.311: [sbi] INFO: NF Service [nudm-uecm] (../lib/sbi/context.c:1436)
open5gs_5gc  | 01/24 21:08:47.311: [sbi] INFO: NF Service [nudm-sdm] (../lib/sbi/context.c:1436)
open5gs_5gc  | 01/24 21:08:47.311: [sbi] INFO: nghttp2_server() [http://127.0.0.12]:7777 (../lib/sbi/nghttp2-server.c:395)
open5gs_5gc  | 01/24 21:08:47.312: [app] INFO: UDM initialize...done (../src/udm/app.c:31)
open5gs_5gc  | 01/24 21:08:47.312: [nrf] INFO: [c3b6a574-bafc-41ee-a98a-05b3ee33cb42] NF registered [Heartbeat:10s] (../src/nrf/nf-sm.c:190)
open5gs_5gc  | 01/24 21:08:47.313: [sbi] INFO: [c3b6a574-bafc-41ee-a98a-05b3ee33cb42] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc  | 01/24 21:08:47.313: [sbi] INFO: [c3b6a574-bafc-41ee-a98a-05b3ee33cb42] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc  | 01/24 21:08:47.313: [sbi] INFO: [c3b6a574-bafc-41ee-a98a-05b3ee33cb42] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc  | 01/24 21:08:47.313: [sbi] INFO: [c3b6a574-bafc-41ee-a98a-05b3ee33cb42] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc  | 01/24 21:08:47.313: [sbi] INFO: [c3b6a574-bafc-41ee-a98a-05b3ee33cb42] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc  | 01/24 21:08:47.313: [sbi] INFO: [c3b6a574-bafc-41ee-a98a-05b3ee33cb42] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc  | 01/24 21:08:47.313: [sbi] WARNING: [c3b6a574-bafc-41ee-a98a-05b3ee33cb42] (NRF-notify) NF has already been added (../lib/sbi/nnrf-handler.c:636)
open5gs_5gc  | 01/24 21:08:47.313: [sbi] INFO: [c3b6a574-bafc-41ee-a98a-05b3ee33cb42] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc  | 01/24 21:08:47.313: [sbi] INFO: [c3b6a574-bafc-41ee-a98a-05b3ee33cb42] NF registered [Heartbeat:10s] (../lib/sbi/nf-sm.c:214)
open5gs_5gc  | 01/24 21:08:47.313: [sbi] WARNING: NF EndPoint updated [127.0.0.12:80] (../lib/sbi/context.c:1618)
open5gs_5gc  | 01/24 21:08:47.313: [sbi] WARNING: NF EndPoint updated [127.0.0.12:7777] (../lib/sbi/context.c:1527)
open5gs_5gc  | 01/24 21:08:47.313: [sbi] INFO: [c3b6a574-bafc-41ee-a98a-05b3ee33cb42] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc  | 01/24 21:08:47.313: [sbi] INFO: [c3b6a574-bafc-41ee-a98a-05b3ee33cb42] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc  | Open5GS daemon v2.6.1
open5gs_5gc  | 
open5gs_5gc  | 01/24 21:08:47.367: [app] INFO: Configuration: 'open5gs-5gc.yml' (../lib/app/ogs-init.c:126)
open5gs_5gc  | 01/24 21:08:47.369: [dbi] INFO: MongoDB URI: 'mongodb://127.0.0.1/open5gs' (../lib/dbi/ogs-mongoc.c:130)
open5gs_5gc  | 01/24 21:08:47.369: [sbi] INFO: NF Service [npcf-am-policy-control] (../lib/sbi/context.c:1436)
open5gs_5gc  | 01/24 21:08:47.369: [sbi] INFO: NF Service [npcf-smpolicycontrol] (../lib/sbi/context.c:1436)
open5gs_5gc  | 01/24 21:08:47.369: [sbi] INFO: NF Service [npcf-policyauthorization] (../lib/sbi/context.c:1436)
open5gs_5gc  | 01/24 21:08:47.369: [sbi] INFO: nghttp2_server() [http://127.0.0.13]:7777 (../lib/sbi/nghttp2-server.c:395)
open5gs_5gc  | 01/24 21:08:47.369: [app] INFO: PCF initialize...done (../src/pcf/app.c:31)
open5gs_5gc  | 01/24 21:08:47.370: [nrf] INFO: [c3bf4fa8-bafc-41ee-bbf9-ddd61cfeabde] NF registered [Heartbeat:10s] (../src/nrf/nf-sm.c:190)
open5gs_5gc  | 01/24 21:08:47.370: [sbi] INFO: [c3bf4fa8-bafc-41ee-bbf9-ddd61cfeabde] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc  | 01/24 21:08:47.370: [sbi] INFO: [c3bf4fa8-bafc-41ee-bbf9-ddd61cfeabde] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc  | 01/24 21:08:47.370: [sbi] INFO: [c3bf4fa8-bafc-41ee-bbf9-ddd61cfeabde] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc  | 01/24 21:08:47.370: [sbi] INFO: [c3bf4fa8-bafc-41ee-bbf9-ddd61cfeabde] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc  | 01/24 21:08:47.371: [sbi] INFO: [c3bf4fa8-bafc-41ee-bbf9-ddd61cfeabde] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc  | 01/24 21:08:47.371: [sbi] INFO: [c3bf4fa8-bafc-41ee-bbf9-ddd61cfeabde] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc  | 01/24 21:08:47.371: [sbi] INFO: [c3bf4fa8-bafc-41ee-bbf9-ddd61cfeabde] NF registered [Heartbeat:10s] (../lib/sbi/nf-sm.c:214)
open5gs_5gc  | Open5GS daemon v2.6.1
open5gs_5gc  | 
open5gs_5gc  | 01/24 21:08:47.412: [app] INFO: Configuration: 'open5gs-5gc.yml' (../lib/app/ogs-init.c:126)
open5gs_5gc  | 01/24 21:08:47.413: [sbi] INFO: NF Service [nnssf-nsselection] (../lib/sbi/context.c:1436)
open5gs_5gc  | 01/24 21:08:47.413: [sbi] INFO: nghttp2_server() [http://127.0.0.14]:7777 (../lib/sbi/nghttp2-server.c:395)
open5gs_5gc  | 01/24 21:08:47.413: [app] INFO: NSSF initialize...done (../src/nssf/app.c:31)
open5gs_5gc  | 01/24 21:08:47.414: [nrf] INFO: [c3c63426-bafc-41ee-9aa5-536a4c27f098] NF registered [Heartbeat:10s] (../src/nrf/nf-sm.c:190)
open5gs_5gc  | 01/24 21:08:47.415: [sbi] INFO: [c3c63426-bafc-41ee-9aa5-536a4c27f098] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc  | 01/24 21:08:47.415: [sbi] INFO: [c3c63426-bafc-41ee-9aa5-536a4c27f098] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc  | 01/24 21:08:47.415: [sbi] INFO: [c3c63426-bafc-41ee-9aa5-536a4c27f098] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc  | 01/24 21:08:47.415: [sbi] INFO: [c3c63426-bafc-41ee-9aa5-536a4c27f098] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc  | 01/24 21:08:47.415: [sbi] INFO: [c3c63426-bafc-41ee-9aa5-536a4c27f098] NF registered [Heartbeat:10s] (../lib/sbi/nf-sm.c:214)
open5gs_5gc  | Open5GS daemon v2.6.1
open5gs_5gc  | 
open5gs_5gc  | 01/24 21:08:47.465: [app] INFO: Configuration: 'open5gs-5gc.yml' (../lib/app/ogs-init.c:126)
open5gs_5gc  | 01/24 21:08:47.466: [sbi] INFO: NF Service [nbsf-management] (../lib/sbi/context.c:1436)
open5gs_5gc  | 01/24 21:08:47.466: [sbi] INFO: nghttp2_server() [http://127.0.0.15]:7777 (../lib/sbi/nghttp2-server.c:395)
open5gs_5gc  | 01/24 21:08:47.466: [app] INFO: BSF initialize...done (../src/bsf/app.c:31)
open5gs_5gc  | 01/24 21:08:47.467: [nrf] INFO: [c3ce46ca-bafc-41ee-bb87-6dd762ab324e] NF registered [Heartbeat:10s] (../src/nrf/nf-sm.c:190)
open5gs_5gc  | 01/24 21:08:47.468: [sbi] INFO: [c3ce46ca-bafc-41ee-bb87-6dd762ab324e] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc  | 01/24 21:08:47.468: [sbi] INFO: [c3ce46ca-bafc-41ee-bb87-6dd762ab324e] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc  | 01/24 21:08:47.468: [sbi] INFO: [c3ce46ca-bafc-41ee-bb87-6dd762ab324e] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc  | 01/24 21:08:47.468: [sbi] INFO: [c3ce46ca-bafc-41ee-bb87-6dd762ab324e] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc  | 01/24 21:08:47.468: [sbi] INFO: [c3ce46ca-bafc-41ee-bb87-6dd762ab324e] NF registered [Heartbeat:10s] (../lib/sbi/nf-sm.c:214)
open5gs_5gc  | Open5GS daemon v2.6.1
open5gs_5gc  | 
open5gs_5gc  | 01/24 21:08:47.525: [app] INFO: Configuration: 'open5gs-5gc.yml' (../lib/app/ogs-init.c:126)
open5gs_5gc  | 01/24 21:08:47.528: [dbi] INFO: MongoDB URI: 'mongodb://127.0.0.1/open5gs' (../lib/dbi/ogs-mongoc.c:130)
open5gs_5gc  | 01/24 21:08:47.528: [sbi] INFO: NF Service [nudr-dr] (../lib/sbi/context.c:1436)
open5gs_5gc  | 01/24 21:08:47.529: [sbi] INFO: nghttp2_server() [http://127.0.0.20]:7777 (../lib/sbi/nghttp2-server.c:395)
open5gs_5gc  | 01/24 21:08:47.529: [app] INFO: UDR initialize...done (../src/udr/app.c:31)
open5gs_5gc  | 01/24 21:08:47.530: [nrf] INFO: [c3d78ec4-bafc-41ee-a8f9-9fa73b4df874] NF registered [Heartbeat:10s] (../src/nrf/nf-sm.c:190)
open5gs_5gc  | 01/24 21:08:47.530: [sbi] INFO: [c3d78ec4-bafc-41ee-a8f9-9fa73b4df874] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc  | 01/24 21:08:47.530: [sbi] INFO: [c3d78ec4-bafc-41ee-a8f9-9fa73b4df874] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc  | 01/24 21:08:47.530: [sbi] INFO: [c3d78ec4-bafc-41ee-a8f9-9fa73b4df874] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc  | 01/24 21:08:47.530: [sbi] INFO: [c3d78ec4-bafc-41ee-a8f9-9fa73b4df874] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc  | 01/24 21:08:47.531: [sbi] INFO: [c3d78ec4-bafc-41ee-a8f9-9fa73b4df874] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc  | 01/24 21:08:47.531: [sbi] INFO: [c3d78ec4-bafc-41ee-a8f9-9fa73b4df874] NF registered [Heartbeat:10s] (../lib/sbi/nf-sm.c:214)
open5gs_5gc  | 01/24 21:08:47.531: [sbi] INFO: [c3d78ec4-bafc-41ee-a8f9-9fa73b4df874] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc  | 01/24 21:08:49.618: [smf] WARNING: PFCP[REQ] has already been associated [127.0.0.7]:8805 (../src/smf/pfcp-sm.c:217)
open5gs_5gc  | 01/24 21:08:49.618: [upf] WARNING: PFCP[RSP] has already been associated [127.0.0.4]:8805 (../src/upf/pfcp-sm.c:213)

**Reply by pgawlowicz** [reply]
The warnings are OK, can you start the gnb and see if it connects?

**Reply by pgawlowicz** [reply]
Could you try to start the docker service with the new docker compose version (2):
```
docker compose up 5gc
```
Instead of version 1:
```
docker-compose up 5gc
```
